src/main/java/com/github/princesslana/eriscasper/data/Data.java

```diff
@@ -35,4 +38,16 @@ public static String toJsonString(Object obj) throws DataException {
       throw new DataException(e);
     }
   }
+
+  public static String toQueryString(Object obj) {
+    JsonNode node = jackson.valueToTree(obj);
+    Map<String, String> queryMapper = new HashMap<>();
+    node.fields()
+        .forEachRemaining(entry -> queryMapper.put(entry.getKey(), entry.getValue().asText()));
+    return queryMapper
+        .entrySet()
```

Comment:
When take a stream of entries, convert to a map, and then back to entries?

Reply:
Yep, that's what I thought of, commit coming in a minute to address that exact point.